easy-ui-react/src/EmptyStateCard/EmptyStateCard.module.scss

```diff
@@ -0,0 +1,48 @@
+@use "mixins" as EmptyStateCard;
+
+.container {
+  max-width: 695px;
+  position: relative;
+}
+
+.inlineMargin {
+  margin: 0 auto;
+}
+
+.decorative {
+  position: relative;
+  display: none;
```

Comment:
this is mainly to account for not having responsive specs and the decorative background relying on "magic" numbers for now